api-wrapper/services/api-wrapper/src/test/java/net/catenax/edc/apiwrapper/security/BasicAuthenticationServiceTest.java

```diff
@@ -1,59 +1,80 @@
+
+import org.powermock.reflect.Whitebox;
 
 public class BasicAuthenticationServiceTest {
 
+    //Correct with new things
+
     private final Monitor monitor = mock(Monitor.class);
-    private final Map<String, String> users = Map.of("hello", "myPassword");
+    private final Vault vault = mock(Vault.class);
+    private final List<BasicAuthVaultLabels> config = List.of(
+            new BasicAuthVaultLabels("usera", "api-basic-auth-usera"),
+            new BasicAuthVaultLabels("userb", "api-basic-auth-userb")
+    );
 
-    private final BasicAuthenticationService authenticationServiceTest = new BasicAuthenticationService(monitor, users);
+    private final BasicAuthenticationService authenticationServiceTest = new BasicAuthenticationService(monitor, vault, config);
 
     @Test
-    void isAuthenticated_shouldReturnNoAuthenticationHeaderSpecified() {
-
-        Map<String, List<String>> map = Map.of("Authorization", new ArrayList<>());
-        assertThat(authenticationServiceTest.isAuthenticated(map)).isEqualTo(false);
+    void CredentialsHaveNoAuthorizationLabel(){
+        Map<String, List<String>> mapCorrectPassword = Map.of("user", new ArrayList<>(Arrays.asList("user dXNlcjpwYXNzd29yZA==", "blablabla")));
+        assertThat(authenticationServiceTest.isAuthenticated(mapCorrectPassword)).isEqualTo(false);
     }
 
     @Test
-    void isAuthenticated_shouldReturnHeaderFormatNotSupported() {
-
-        Map<String, List<String>> map1authorizationComponent = Map.of("Authorization", new ArrayList<>(Arrays.asList("Only1Compoment")));
-        Map<String, List<String>> map3authorizationComponents = Map.of("Authorization", new ArrayList<>(Arrays.asList("First", "2nd", "3")));
-
-        assertThatThrownBy(() -> authenticationServiceTest.isAuthenticated(map1authorizationComponent)).isInstanceOf(IllegalArgumentException.class);
-        assertThatThrownBy(() -> authenticationServiceTest.isAuthenticated(map3authorizationComponents)).isInstanceOf(IllegalArgumentException.class);
+    void AuthorizationHeaderIsNotValidToken(){
+        Map<String, List<String>> mapCorrectPassword = Map.of("Authorization", new ArrayList<>(Arrays.asList("userdXNlcjpwYXNzd29yZA==")));
+        assertThat(authenticationServiceTest.isAuthenticated(mapCorrectPassword)).isEqualTo(false);
     }
 
     @Test
-    void isAuthenticated_shouldReturnIncorrectlylyDecodedAuthorizationHeaderNotSupported() {
-        Map<String, List<String>> mapUnsupported = Map.of("Authorization", new ArrayList<>(Arrays.asList("user password", "blablabla")));
-
-        assertThatThrownBy(() -> authenticationServiceTest.isAuthenticated(mapUnsupported)).isInstanceOf(IllegalArgumentException.class);
+    void shouldReturnIsAuthenticated_shouldReturnNoAuthenticationHeaderSpecified() throws Exception {
+        Result<BasicAuthenticationService.BasicAuthCredentials> result = Whitebox.invokeMethod(authenticationServiceTest,"decodeAuthHeader","userdXNlcjpwYXNzd29yZA");
```

Comment:
You don't like to set spaces or? 😅
Think it's easier if you just run reformat in intellij for this on the whole file.

```suggestion
        Result<BasicAuthenticationService.BasicAuthCredentials> result = Whitebox.invokeMethod(authenticationServiceTest, "decodeAuthHeader", "userdXNlcjpwYXNzd29yZA");
```